backend/backend/services/graph/subgraphs.py

```diff
@@ -20,13 +20,22 @@ def __init__(self, protocol, chain):
         self.protocol = protocol
         self.chain = chain
         if protocol == "sporkdao-token":
-            self.deployments = {"base": "sporkdao-token"}
+            self.deployments = {"base": protocol}
             self.service_type = "hosted-service"
             self.query_id = protocol
             self.template_file_location = f"subgraphs_custom/{protocol}/subgraph.yaml"
             self.schema_file_location = f"subgraphs_custom/{protocol}/schema.graphql"
             self.mappers = f"subgraphs_custom/{protocol}/src/sporkdao.ts"
             return
+        # TODO
```

Comment:
What is the TODO here? Is it to abstract away the logic after the 'if' statement? Because it is repetitive so we should abstract it.

Reply:
since we dont have the source code for this subgraph, we do not have the manifest/mapping/schema files (yet)